Maths/FindMax.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Function to find the maximum number given an array of integers
+ * Returns the maximum number of the array
+ * If the array is empty it returns the string 'Array is empty'
+ */
+
+export const findMax = (arr) => {
+  if (arr.length === 0) { return 'Array is empty' }
```

Comment:
Throw an error, not a string.

Reply:
Agree with @raklaptudirm. Whenever you know the conditions where your function will fail (I call them nice errors) then throw then instead of handling them yourself (by returning a string) throw an error instead so that the user can handled them according to their business rules.